src/app/contact-page/contact-page.component.css

```diff
@@ -0,0 +1,17 @@
+#form-container {
+  margin: 0 auto;
+  width: 60%;
+  background-color: #fff;
+  padding: 40px;
+  margin-top: 30px;
+}
+
+#button-submit-ticket {
+  border-radius: 0%;
```

Comment:
Not sure why you removed the border radius here. Are rounded borders bad for buttons?